services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -44,13 +43,23 @@ export class HookCallbackHandlerService {
       );
     }
 
+    if (processName === 'website-monitoring') {
```

Comment:
Use an enum over a string literal.

Reply:
P1

Reply:
Sounds like overkill, there just 2 cases.

Reply:
It's unrelated to quantity. It's about not being able to compare against values that don't exist or make typos.

Reply:
![image](https://github.com/ballerine-io/ballerine/assets/23522869/6a03dd7e-4fef-42f4-9bae-b78a5e0c2151)
I doubt that you can make typo when variable has type.